src/main/java/org/jabref/logic/bibtex/comparator/EntryComparator.java

```diff
@@ -65,10 +66,10 @@ public int compare(BibEntry e1, BibEntry e2) {
         // sorted according to last name.
         if (InternalBibtexFields.getFieldProperties(sortField).contains(FieldProperty.PERSON_NAMES)) {
             if (f1 != null) {
-                f1 = AuthorList.fixAuthorForAlphabetization((String) f1).toLowerCase();
+                f1 = AuthorList.fixAuthorForAlphabetization((String) f1).toLowerCase(Locale.ROOT);
```

Comment:
Is there any issue solved with this?

Is it worth that we spend two hours on this? Why?